Navigation/Signoffs/Livestream Camera Subsystem.md

```diff
@@ -0,0 +1,114 @@
+# Livestream Camera Subsystem
+# Function of the subsystem
+
+The Camera Subsystem of the Autonomous Crawlspace Inspection Robot is to capture real-time video footage of the crawlspace and transmit it to a central control unit for immediate viewing and assessment.
+
+
+# Constraints:
+
+## Constraint 1:
+The camera must be connected to a network with a stable and unrestricted bandwidth of at least 78.64 Mbps to ensure seamless video streaming without buffering or reduced video quality.
+
+### information about Constraint 1:
```

Comment:
Couldn't we transmit at lower resolution on less bandwidth?

Reply:
What resolution is necessary? What frame rate is necessary? Without knowing these where are you drawing these numbers from?